src/vhost_user/message.rs

```diff
@@ -459,6 +479,49 @@ impl VhostUserMsgValidator for VhostUserMemoryRegion {
 /// Payload of the VhostUserMemory message.
 pub type VhostUserMemoryPayload = Vec<VhostUserMemoryRegion>;
 
+/// Single memory region descriptor as payload for ADD_MEM_REG and REM_MEM_REG
+/// requests.
+#[repr(packed)]
+#[derive(Default, Clone, Copy)]
```

Comment:
`#[repr(packed)]` is wrong. I checked the C code there it's not defined as packed there so this should be `#[repr(C)]` (this might have no impact on the actual layout but it is more correct.)

Reply:
Sounds good!